xml/installation-installation-configure_lbaas.xml

```diff
@@ -232,482 +233,363 @@
-| id                  | 3ce608bf-8835-4638-871d-0efe8ebf55ef |
-| port_id             | 7e5e0038-88cf-4f97-a366-b58cd836450e |
-| router_id           | 6aafc9a9-93f6-4d7e-94f2-3068b034b823 |
-| status              | ACTIVE                               |
-| tenant_id           | 4b31d0508f83437e83d8f4d520cda22f     |
-+---------------------+--------------------------------------+</screen>
-   </step>
-   <step>
-    <para>
-     Show second Floating IP.
-    </para>
-<screen>&prompt.ardana;openstack floating ip show 680c0375-a179-47cb-a8c5-02b836247444
-+---------------------+--------------------------------------+
-| Field               | Value                                |
-+---------------------+--------------------------------------+
-| fixed_ip_address    | 10.247.94.133                        |
-| floating_ip_address | 10.247.96.27                         |
-| floating_network_id | d3cb12a6-a000-4e3e-82c4-ee04aa169291 |
-| id                  | 680c0375-a179-47cb-a8c5-02b836247444 |
-| port_id             | ca95cc24-4e8f-4415-9156-7b519eb36854 |
-| router_id           | 6aafc9a9-93f6-4d7e-94f2-3068b034b823 |
-| status              | ACTIVE                               |
-| tenant_id           | 4b31d0508f83437e83d8f4d520cda22f     |
-+---------------------+--------------------------------------+</screen>
-   </step>
-   <step>
-    <para>
-     <command>Ping</command> first Floating IP.
-    </para>
-<screen>&prompt.ardana;ping -c 1 10.247.96.26
-PING 10.247.96.26 (10.247.96.26) 56(84) bytes of data.
-64 bytes from 10.247.96.26: icmp_seq=1 ttl=62 time=3.50 ms
+    <screen><?dbsuse-fo font-size="0.70em"?>&prompt.ardana;openstack keypair create lb_kp1 &gt; lb_kp1.pem
+chmod 400 lb_kp1.pem
 
---- 10.247.96.26 ping statistics ---
-1 packets transmitted, 1 received, 0% packet loss, time 0ms
-rtt min/avg/max/mdev = 3.505/3.505/3.505/0.000 ms</screen>
-   </step>
-   <step>
-    <para>
-     <command>Ping</command> second Floating IP.
+ls -la lb*
+-r-------- 1 ardana ardana 1680 May 13 12:52 lb_kp1.pem</screen>
+    <para>
+      Verify keypair list
+    </para>
+    <screen><?dbsuse-fo font-size="0.70em"?>&prompt.ardana;openstack keypair list
++---------------+-------------------------------------------------+
+| Name          | Fingerprint                                     |
++---------------+-------------------------------------------------+
+| id_rsa2       | d3:82:4c:fc:80:79:db:94:b6:31:f1:15:8e:ba:35:0b |
+| lb_kp1        | 78:a8:0b:5b:2d:59:f4:68:4c:cb:49:c3:f8:81:3e:d8 |
++---------------+-------------------------------------------------+</screen>
+   </step>
+   <step>
+    <para>
+     Get image name to boot and boot two VMs
+    </para>
+<screen><?dbsuse-fo font-size="0.70em"?>&prompt.ardana;openstack image list | grep cirros
+| faa65f54-e38b-43dd-b0db-ae5f5e3d9b83 | cirros-0.4.0-x86_64             | active |</screen>
+    <para>
+      Boot a VM that will host the "fake" webserver"
```

Comment:
```suggestion
      Boot a VM that will host the fake webserver:

Reply:
Instead of fake, I'd probably go with "psuedo" or "copy" or even "imitation"

Reply:
_Pseudo_ has my vote.

Reply:
Pseudo works.